src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/InputJavadocContentLocationTrailingSpace.java

```diff
@@ -0,0 +1,82 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoccontentlocation;
+
+/**
+ * config:  default
+ * location="SECOND_LINE"
+ * NOTE: This test input contains trailing spaces.
+ * Please make sure your IDE does not delete them when saving.
+*/
+public interface InputJavadocContentLocationTrailingSpace {
+
+    /**  
+     * ^^ There is a trailing space in the first line // OK
+     */
+    void space();
+
+    /**********************
+     * Trailing asterisks in the first line // OK
+     *
+     **********************/
+    void asterisk();
+
+    /** ******
+     * There is a space, then leading asterisks in the first line // OK
+     */
+    void spaceAsterisk();
+
+    /**********************              
+     * Trailing asterisks, then spaces ^^ in the first line // OK
+     *
+     **********************/
+    void asteriskSpace();
+
+    /**      ***************************************            
+     * There is a spaces, then leading asterisks, then spaces ^^ in the first line // OK
+     */
+    void spaceAsteriskSpace();
+
+    /*** **
+     * There is an extra asterisk after the Javadoc start,
+     * then a space, then leading asterisks in the first line // OK
+     * The generated javadoc will be "There is an extra ..."
+     */
+    void asteriskSpaceAsterisk();
+
+    /**   *  ***
+     * There is a space, then a leading asterisk, then a space,
+     * then asterisks in the first line // OK
+     * The generated javadoc will be "*** There is a space ..."
+     * Weird corner case, we do not report a violation here due to the parser limitation:
```

Comment:
Does this still need to be a corner case now that we aren't using the parser?

Reply:
Done.